tests/unit/models/test_volume.py

```diff
@@ -50,10 +50,10 @@ def test_volume_valid():
     )
     assert volume.volume_schema == "gpt"
     assert len(volume.structure) == 3
-    assert volume.structure[0].size == 6442450944
+    assert volume.structure[0].size == 3221225472
```

Comment:
It's not particularly obvious where this number came from. What about:

```suggestion
    assert volume.structure[0].size == 3 * 1024**3
```

(Similar for below)